metaflow/plugins/argo/argo_workflows.py

```diff
@@ -867,11 +873,35 @@ def _container_templates(self):
                 if v is not None and k not in env_vars_to_skip
             }
 
+            # Tmpfs variables
+            use_tmpfs = resources["use_tmpfs"]
+            tmpfs_size = resources["tmpfs_size"]
```

Comment:
I'm guess the memory backing `tmpfs` is part of the total memory assigned to the pod, right? 

What happens if a user accidentally sets tmpfs_size >= total memory of the pod? Does K8s fail with an appropriate error message?

Reply:
with the cluster that is available for my testing, nothing fails even when the emptyDir size is set to exceed the memory available for the pod. free disk space reported on the mount stays at `memory // 2` though, which seems to be a hard cap for that.

Reply:
Excellent! So, looks like internally Linux takes care that `tmpfs` never exceeds half of main memory.